src/core/seo.js

```diff
@@ -14,6 +15,76 @@ export async function run(conf, doc, cb) {
     return; // no abstract, so nothing to do
   }
   insertMetaDescription(firstParagraph);
+  if (conf.doJsonLd) {
```

Comment:
If suggest moving all this into it's own `function addJSONLDInfo(conf) { ... }` instead.

Reply:
done @marcoscaceres, kindly check